app/pactas/models/PostalAddress.java

```diff
@@ -41,17 +42,17 @@ public CountryCode getCountryCode() {
     }
 
     public Address getAddress() {
-        Address address = new Address(getCountryCode());
+        AddressBuilder addressBuilder = AddressBuilder.of(getCountryCode());
         if (addressLine != null) {
-            address.setStreetName(addressLine);
+            addressBuilder.streetName(addressLine);
```

Comment:
`AdressBuilder.streetName` accepts parameters which are `null`: http://sphereio.github.io/sphere-jvm-sdk/javadoc/v1.0.0-M17/io/sphere/sdk/models/AddressBuilder.html#streetName-java.lang.String-

So you can remove the if blocks and have a one-liner.